web/src/main/webapp/loadMenu.ts

```diff
@@ -322,6 +334,34 @@ class CSVFileDialog extends Dialog {
             repeat: 1,
             name: null,
             fileKind: "csv",
+            logFormat: null,
+        };
+    }
+}
+
+class GenericLogDialog extends Dialog {
+    constructor() {
+        super("Load Generic Logs",
+            "Loads log files from all machines that are part of the service.");
+        this.addTextField("folder", "Folder", FieldKind.String, "/",
+            "Folder on the remote machines where all the log files are found.");
+        this.addTextField("fileNamePattern", "File name pattern", FieldKind.String, "*.log",
+            "Shell pattern that describes the names of the files to load.");
+        this.addTextField("logFormat", "Log format", FieldKind.String, "%{SYSLOG}",
```

Comment:
Should we start with this being a dropdown box (addSelectField) with a small number of fixed patterns?
Do you mean for the user to type a full pattern here?

Reply:
@mbudiu-vmw Will add the drop-down box as part of next PR.